docs/volunteering/glossary.md

```diff
@@ -24,11 +24,6 @@ teams build their robots; see also _SRnnnn_
 : the group of volunteers responsible for delivering a particular year's
 _Competition programme_
 
-_Core Team_:
-
-: the group of 8 or so volunteers responsible for delivering the _SR2019_
-_Competition programme_; this has since been replaced by the _Competition Team_
-
```

Comment:
If we're removing this definition please could we also remove all mentions of the term?

Reply:
Looks like there was only one so have removed it :)